src/tmf882x/__init__.py

```diff
@@ -180,6 +198,53 @@ def spad_map(self, map_id: int) -> None:
         with self._configuration_mode():
             self.bus.write_byte_data(self.address, register=0x34, value=map_id)
 
+    @property
+    def i2c_address_change(self) -> int:
+        with self._configuration_mode():
+            return self.bus.read_byte_data(self.address, register=0x3E)
+
+    @i2c_address_change.setter
+    def i2c_address_change(self, value: int) -> None:
+        with self._configuration_mode():
+            self.bus.write_byte_data(self.address, register=0x3E, value=value)
+
+    @property
+    def i2c_address(self) -> int:
+        #i2c address is 7 bit and zero bit should be null
+        #shifting 1 bit and sending 
+        with self._configuration_mode():
+            return self.bus.read_byte_data(self.address, register=0x3B) >> 1
+
+    @i2c_address.setter
+    def i2c_address(self, value: int) -> None:
+        #i2c address is 7 bit checking for valid i2c address
+        if (value < 8) or (value > 127):
+            raise ValueError("I2C address is 7 bit. Min address 0x08 and max address is 0x7F")
+
+        value = value << 1
+        with self._configuration_mode():
+            self.bus.write_byte_data(self.address, register=0x3B, value=value)
+
+    @property
+    def gpio_0(self) -> int:
+        with self._configuration_mode():
+            return self.bus.read_byte_data(self.address, register=0x31)
+
+    @gpio_0.setter
+    def gpio_0(self, value: int) -> None:
+        with self._configuration_mode():
+            self.bus.write_byte_data(self.address, register=0x31, value=value)
+
+    @property
+    def gpio_1(self) -> int:
+        with self._configuration_mode():
+            return self.bus.read_byte_data(self.address, register=0x32)
+
+    @gpio_0.setter
```

Comment:
I think this should be:
```suggestion
    @gpio_1.setter
```